packages/discord.js/src/util/Util.ts

```diff
@@ -130,7 +135,7 @@ class Util extends null {
       codeBlockContent = true,
       inlineCodeContent = true,
     } = {},
-  ) {
+  ): string {
```

Comment:
Can’t this return type be inferred?

Reply:
doesn't seem to like when it's not annotated